pandas/conftest.py

```diff
@@ -223,6 +223,20 @@ def all_compare_operators(request):
     return request.param
 
 
+@pytest.fixture(params=['__pos__', '__neg__', '__inv__', '__invert__',
+                        '__abs__'])
+def all_unary_operators(request):
```

Comment:
can this have a name reflecting the fact that these are the strings and not operator.pos, operator.neg, etc?